metainfo.patch

```diff
@@ -0,0 +1,7 @@
+diff --git i/cinecred.metainfo.xml w/cinecred.metainfo.xml
+index dbf835f..c75dce5 100644
+--- cinecred.metainfo.xml
++++ cinecred.metainfo.xml
+@@ -11 +11 @@
+-  <icon type="stock">cinecred</icon>
++  <icon type="stock">com.cinecred.cinecred</icon>
```

Comment:
this is a desktop-application right ? Then you don't need this tag. Remove it altogether.

Reply:
Done. Thanks for the hint!

Just a quick question: it's only for Flatpak where the icon tag isn't needed since Flatpak adds it by itself during the build process, right? Because the AppStream docs state:

> The `<icon/>` tag describes the component icon. It is mostly used for GUI applications (component-type `desktop-application`).

So to me it looks correct to keep the icon tag in the general upstream metainfo, and remove it only for the Flatpak build (i.e., what we're doing right now :) ).

Reply:
it's talking about catalogue data not metainfo. It's needed for console apps when they have no desktop file.

For desktop-applications: appstream gets the desktop file using `launchable` -> desktop file points to icon -> icon is found if installed.

Reply:
Ah, I see. Then I'll remove the `<icon>` tag upstream in the next release, which will allow me to also get rid of this patch. Thanks!